va/http_test.go

```diff
@@ -517,6 +517,31 @@ func httpTestSrv(t *testing.T) *httptest.Server {
 			http.StatusMovedPermanently)
 	})
 
+	mux.HandleFunc(fmt.Sprintf("/max-redirect"),
+		func(resp http.ResponseWriter, req *http.Request) {
+			http.Redirect(
+				resp,
+				req,
+				fmt.Sprintf("http://example.com:%d/1/max-redirect", httpPort),
+				http.StatusMovedPermanently,
+			)
+		})
+	// A path that sequentually redirects, creating an incrementing redirect
+	// that will terminate when the redirect limit is reached and ensures each
+	// URL is different than the last.
+	for i := 1; i <= maxRedirect+1; i++ {
+		s, x := strconv.Itoa(i), strconv.Itoa(i+1)
```

Comment:
I think you can delete this line and instead use `%d` in your string formats below, like:
```
mux.HandleFunc(fmt.Sprintf("/%d/max-redirect", i),
```
on the next line and similarly
```
fmt.Sprintf("http://example.com:%d/%d/max-redirect", httpPort, i+1),
```
on line 539.

Reply:
Thanks, I had it that way first and the test cases did not work. I'll look into it more to figure out exactly why.

Reply:
Pushed a fix for this by re-assigning i in the loop. Thanks!